lib/api/BackbeatServer.js

```diff
@@ -30,18 +30,20 @@ class BackbeatServer {
     /**
      * Last log message to send per incoming request
      * @param {werelogs.newRequestLogger} logger - request logger
+     * @param {object} data - if error, can carry a specific error message
      * @param {object} req - request object
      * @param {object} res - response object
      * @return {undefined}
      */
-    _logRequestEnd(logger, req, res) {
+    _logRequestEnd(logger, data, req, res) {
```

Comment:
Since `data` is optional and `req` and `res` are mandatory params, it would be less intrusive and more appropriate to put it as last argument, and `data` looks somewhat too generic for a variable name here.

Reply:
hmm, for now i'll name it `desc`. I want this field to be a more informative message, so I added this field :)